src/kili/services/export/format/kili/__init__.py

```diff
@@ -265,31 +292,49 @@ def _scale_normalized_vertices_pdf_annotation(
 
 
 def _scale_normalized_vertices_image_video_annotation(
-    annotation: Dict, asset: Dict, is_label_rotated: bool
+    annotation: Dict,
+    asset: Dict,
+    _is_label_rotated: bool,
+    rotation: int,
+    _normalized_vertices: Optional[bool],
```

Comment:
Make `_normalized_vertices` a bool with a default value if you want to make it optional for the caller. The code will be less subject to errors like checking the value is exactly False or an just a falsy value.

Reply:
I see, still you can update internal methods to have a nicer api.